<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="openSecondWindow_button">
      <property name="text">
       <string>ОТКРЫТЬ ВТОРОЕ ОКНО</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="sound_menu">
     <property name="title">
      <string>Звук</string>
     </property>
     <addaction name="soundOn_action"/>
     <addaction name="soundOff_action"/>
    </widget>
    <addaction name="sound_menu"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="about_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="soundOn_action">
   <property name="text">
    <string>Вкл.</string>
   </property>
  </action>
  <action name="soundOff_action">
   <property name="text">
    <string>Выкл.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
